conda-store-server/conda_store_server/action/download_packages.py

```diff
@@ -1,10 +1,24 @@
+import os
 import pathlib
 import shutil
 import typing
 
+# This import is needed to avoid the following error on conda imports:
+# AttributeError: 'Logger' object has no attribute 'trace'
```

Comment:
Yes, this is really annoying! I've been meaning to fix that upstream for a while. There should be no global monkey patching of `logging` _just_ to add a utility function 😠

Reply:
FYI: It was discussed before in the context of a different issue and received a "won't fix" reply: https://github.com/conda/conda/issues/8919#issuecomment-510912585. So it might depend on what APIs are considered private in conda.

Reply:
Yes, that was a different team in a different time. We will get this fixed in 2024, hopefully.